File: Java/1074.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  public static void main(String[] args){
    int teste, i, num;
    Scanner sc = new Scanner(System.in);
    teste = sc.nextInt();
    
    for( i = 1; i <= teste; i++){
      num = sc.nextInt();
      if(num == 0){
        System.out.printf("NULL");
      }
      else if(num % 2 == 0){
        if(num > 0){
        System.out.printf("EVEN POSITIVE");
        }
        else{
          System.out.printf("EVEN NEGATIVE");
        }
      }
      else{
        if(num > 0){
          System.out.printf("ODD POSITIVE");
        }else{
          System.out.printf("ODD NEGATIVE");
        }
      }
    }        
  }
 
}

File: C/1074.c
#include <stdio.h>
int teste,i,num;
main(){
    scanf("%d",&teste);
    for(i=0; i<teste ; i++){
        scanf("%d",&num);
        if(num!=0){
            if(abs(num)%2==0 && num>0){
                printf("EVEN POSITIVE\n");}
            if(abs(num)%2==0 && num<0){
                printf("EVEN NEGATIVE\n");}
            if(abs(num)%2!=0 && num<0){
                printf("ODD NEGATIVE\n");}
            if(abs(num)%2!=0 && num>0){
                printf("ODD POSITIVE\n");}}
        else{printf("NULL\n");}}
    return 0;
}